learn/guides/ballerina-shell.md

```diff
@@ -151,6 +164,25 @@ The function can in turn be called and assigned to a variable.
 30
 ```
 
+###  Defining Types
+
+Type definitions can be written same as a function.
```

Comment:
```suggestion
Type definitions can be written the same as a function.
```

Reply:
Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75